android/src/main/java/com/rssignaturecapture/RSSignatureCaptureView.java

```diff
@@ -68,8 +68,8 @@ public RSSignatureCaptureView(Context context, SignatureCallback callback) {
 		mPaint.setStrokeCap(Paint.Cap.ROUND);
 		mPaint.setStrokeJoin(Paint.Join.ROUND);
 
-		mMinWidth = convertDpToPx(8);
-		mMaxWidth = convertDpToPx(16);
+		mMinWidth = convertDpToPx(4);
```

Comment:
why did you change this values?

Reply:
Has @agrass said :
> Also I'm changing the default stroke width of Android because is different of the default stroke of IOS.

Reply:
ok cool

Reply:
Yes I changed to have similar stroke width in Android and IOS. what do you think? Anyway in other pull request it would be possible to set both with a param

Reply:
@agrass It's better like this, the signature looks more "professional" in my opinion.

Reply:
@jedt can you merge this ?

Reply:
Any update about this? Is needed another change?